### PR TITLE
Commenting Code and Front End Changes

### PR DESCRIPTION
**Pull request by lelekaz**

### CODE REVIEW

File: views/aboutus/index.ejs [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <div class="row col-12"><%- include('../partials/navbar'); %></div>
 
       <div class="row col-12">
-        <div class="w-50 m-auto mt-5 mb-5" style="border-radius: 15px">
```
Comment (sderacy) [inline views/aboutus/index.ejs:14]: I don't oppose changing the width, but why were the margins removed? They were there to balance the amount of blank space above and below the content:
image
image

**Review by sderacy — COMMENTED** [review]
Looks good, I plan on adding a couple of comments too

File: views/aboutus/index.ejs (hunk 1 above)
Comment (lelekaz) [reply views/aboutus/index.ejs:14]: Oh okay, I'll put them back. I don't remember why I removed them.

Comment (sderacy) [reply views/aboutus/index.ejs:14]: No worries, I didn't mind them being gone but I wanted to know how it was better than what was there originally. I like the way the text wraps much more now though.

File: views/aboutus/main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    // Otherwise, make the width 50vw
+  } else {
+    outer_div.classList.add("w-50");
+    outer_div.classList.remove("w-75");
+  }
+}
+
+var media_query = window.matchMedia("(max-width: 900px)");
+changeWidthDiv(media_query); // Call listener function at run time
+media_query.addListener(changeWidthDiv); // Attach listener function on state changes
```
Comment (sderacy) [inline views/aboutus/main.js:18]: Just a heads up that line 18 calls a deprecated function.

**Comment by sderacy** [comment]
@paytonshaltis I went through and added comments for most things and I think Leah and I might've fixed the margins. 

The last thing consistency-wise is this:

image

When you switch to light mode after having high contrast, this and the footer don't immediately update. When you click save changes, it looks fine, but it's for that brief moment that it just looks a little weird. I forget how the preview works but is this an easy fix?

**Comment by paytonshaltis** [comment]
> The last thing consistency-wise is this:
> When you switch to light mode after having high contrast, this and the footer don't immediately update. When you click save changes, it looks fine, but it's for that brief moment that it just looks a little weird. I forget how the preview works but is this an easy fix?

I'm not totally sure what you're referring to... I don't see the footer in your screenshot, and the rest of the picture looks fine to me. Could you elaborate?

If it turns out to be something with the preview, I don't think it will be too bad, but I'll look into it once I have more info.

**Review by paytonshaltis — APPROVED** [review]
I approve of my Goon Squad 😤

**Comment by sderacy** [comment]
> The function `displayChanges()` in this file has to have the exact same code as `defaultSettings.ejs` does, and it looks like you added code to fix the heart and logo in the first but not the second file @sderacy or whoever else touched it.

That makes a lot of sense actually. I'm assuming I'm the one who modified it so thank you for clearing that up. If we have to make any more changes I'll try to remember to edit both 😅 

Nice work everyone!

**Pull request merged**